style/components/list.css

```diff
@@ -0,0 +1,57 @@
+/* single-lined-list  */
+.list{
+    font-size: 1rem;
+    font-weight: 400;
+    padding: 8px 0;
+    line-height: 1.75rem;
+    border-radius: 0px;
+    max-width: 600px;
+    background-color: #fff;
+    border: 1px solid rgba(0,0,0,.1);
+}
+.list-items{
+    display: flex;
+    align-items: flex-start;
+    justify-content: center;
+    padding: 0 16px;
+    height: 48px;
+    cursor: pointer;
+    flex-direction: column;
+    user-select: none;
+}
+.list-width{
+    height: 72px;
+}
+.list-items:hover{
+    background-color: rgba(0,0,0,.1);
+    border-radius: 0px;
+}
+.list-items:active{
+    background-color: rgba(0,0,0,0.3);
+    border-radius: 0px;
+    outline: none;
+}
+.list-sec-text{
+    margin-top: -8px;
+    font-size: .875rem;
+    font-weight: 400;
+    color: rgba(0, 0, 0,.54);
+}
+.list-with-icons{
+    width: 100%;
+    display: flex;
+    align-items: center;
+    gap: 32px;
+}
+.list-divider{
+    height: 0px;
+    border-bottom: 1px solid;
+    color: rgba(0, 0, 0,.12);
+}
+.input-ckeckbox{
+    width: 18px;
+    height: 18px;
```

Comment:
use other units(rem) when the value is more than 10px.

Reply:
done!!